### PR TITLE
creating soa-archetype

### PR DESCRIPTION
**Pull request by mjburghoffer**
@scottbessler  @christydennison  @mgrinolds please look at this and review!

### CODE REVIEW

**Comment by scottbessler** [comment]
lgtm!!

File: soa-archetype/src/main/resources/archetype-resources/create-service.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#set($d = "$")
+#set($dbr = "${")
+
+CAMEL="${friendlyName}"
+LOWER="${rootArtifactId}"
+USERNAME=""
+PASSWORD=""
+
+main() {
```
Comment (christydennison) [inline soa-archetype/src/main/resources/archetype-resources/create-service.sh]: You need to add back in the part for getting the username and password:
```
while [ "$(curl -u "${USERNAME}:${PASSWORD}" -s -o /dev/null -w "%{http_code}" https://teamcity.amz.relateiq.com/httpAuth/app/rest/builds)" != "200" ]; do
        echo -n "TC Username: ";
        read USERNAME;
        echo -n "TC Password: ";
        read -s PASSWORD;
    done
```

Comment (christydennison) [inline soa-archetype/src/main/resources/archetype-resources/create-service.sh]: Looks like you just need to call `loadArgs`

Comment (mjburghoffer) [inline soa-archetype/src/main/resources/archetype-resources/create-service.sh]: args don't need to be loaded anymore.  They are set statically from the archetype.

Comment (christydennison) [inline soa-archetype/src/main/resources/archetype-resources/create-service.sh]: You pass in the username and password there?

File: soa-archetype/src/test/resources/projects/first/goal.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+package
```
Comment (christydennison) [inline soa-archetype/src/test/resources/projects/first/goal.txt]: ? Delete?

Comment (mjburghoffer) [inline soa-archetype/src/test/resources/projects/first/goal.txt]: no we need this file

**Comment by christydennison** [comment]
LGTM

**Pull request merged**